code/modules/shuttle/computer.dm

```diff
@@ -57,6 +57,11 @@
 			if(M.mode != SHUTTLE_IDLE)
 				to_chat(usr, "<span class='warning'>Shuttle already in transit.</span>")
 				return
+		if(!(href_list["move"] in params2list(possible_destinations)))
+			to_chat(usr, "<span class='userdanger'>I'm sure your \"Special\" sock would be proud of you for this action.</span>") // thanks citruses

```

Comment:
No.

Reply:
fun police

Reply:
Wsock reference?
Personally don't need client-side hacks to find href exploits like this, nor do I support griefing when it isn't at least funny, which is mostly why I'm trying to find and get abusable hrefs fixed.